tests/test_dockhub.py

```diff
@@ -0,0 +1,44 @@
+from click.testing import CliRunner
+
+from dockhub import main, DOCKERHUB_URL
+
+
+def test_basic():
+    """Verify that it runs and spits out help."""
+    runner = CliRunner()
+    result = runner.invoke(main, "--help")
+    assert result.exit_code == 0
+
+
+def test_missing_dh_username(monkeypatch):
+    """Verify missing DH_USERNAME kicks up error."""
+    monkeypatch.setenv("DH_USERNAME", "")
+    runner = CliRunner()
+    result = runner.invoke(main)
+    assert result.exit_code == 1
+    # Assert that a substring of the error message is in the output
+    assert "variable DH_USERNAME" in result.output
```

Comment:
With this test, we use the pytest `monkeypatch` fixture to set an environment variable. The variable will go back to its original value when this test is done.